File: pytools/data/data.code.lua53.client/AckRoleLoginOkNoRole.lua
AckRoleLoginOkNoRole = {}
setmetatable(AckRoleLoginOkNoRole, {__index = _G})
local _ENV = AckRoleLoginOkNoRole


function new()
	local tab_obj = {}
	setmetatable(tab_obj, {__index = AckRoleLoginOkNoRole})
	tab_obj:init()
	return tab_obj
end

function init(self)
end


function decode(self, pack)
	return self
end

File: pytools/examples/golang/src/client/proto/ack_role_login_ok_no_role.go
package proto

import (
	"packet"
)

type AckRoleLoginOkNoRole struct {
}

func AckRoleLoginOkNoRoleDecode(pack *packet.Packet) *AckRoleLoginOkNoRole {
	ackRoleLoginOkNoRole := &AckRoleLoginOkNoRole{}

	return ackRoleLoginOkNoRole
}
